fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -389,7 +389,16 @@ public Response createOrReplaceObjectRdf(
         }
 
         try {
-            session.save();
+            // prevent SN-Sibling nodes from creating.
+            synchronized (LOGGER) {
```

Comment:
I do not understand why you are locking on anything.


> On Jul 12, 2016, at 11:59 AM, lsitu <notifications@github.com> wrote:
> 
> In fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java:
> 
> > @@ -389,7 +389,16 @@ public Response createOrReplaceObjectRdf(
> >          }
> >  
> >          try {
> > -            session.save();
> > +            // prevent SN-Sibling nodes from creating.
> > +            synchronized (LOGGER) {
> 
> @ajs6f Thanks. I think we can change it to use the FedoraLdp class or create a static object as the lock. What do you suggest?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>